GitCommands/PathUtil.cs

```diff
@@ -89,6 +89,18 @@ public static bool IsUrl(string path)
                  || path.StartsWith("file:", StringComparison.CurrentCultureIgnoreCase));
         }
 
+        public static bool CanBeGitURL(string url)
+        {
+            if (url.IsNullOrWhiteSpace())
+            {
+                return false;
+            }
+
+            return PathUtil.IsUrl(url)
```

Comment:
Does it work without repeating the class?
```suggestion
            return IsUrl(url)
```

Reply:
> Does it work without repeating the class?

Fixed.



> Could you also sign-off the `contributors.txt` file, please? Open another PR for this in order to avoid merge conflicts.

I am already a listed contributor - and I recall signing this several months ago. Do I still need to do this?

Reply:
Opened a dedicated PR for the contributors file. My name wasn't there.. must be new since I last contributed.